src/requests/sign-request.spec.ts

```diff
@@ -182,5 +182,26 @@ describe('create-signature', () => {
 
       assert.ok(Object.values(result).every((h) => typeof h !== 'undefined'))
     })
+
+    it('CRN header is optional', () => {
+      const result = signRequest(VALID_SECRET, VALID_REQUEST, undefined, {
+        appId: 'appId',
+        spaceId: 'spaceId',
+        envId: 'envId',
+      })
+
+      assert.equal(result['x-contentful-crn'], undefined)
+    })
+
+    it('includes CRN header', () => {
```

Comment:
Maybe also check `x-contentful-signed-headers` includes this header it in the list

Reply:
Done https://github.com/contentful/node-apps-toolkit/pull/324/commits/6dce177ad9b5b349f42b83deb647da5177f8df6b